data/background.ts

```diff
@@ -218,6 +218,63 @@ let IMMIGRATED = [{
   "value": 6
 }];
 
+let EMIGRATED_COUNTRY = [{
+  "name": "BELGIUM",
+  "value": 1
+}, {
+  "name": "CHINA",
+  "value": 10
+}, {
+  "name": "ENGLAND",
+  "value": 1
+}, {
+  "name": "INDIA",
+  "value": 4
+}];
+
+let NUM_LANGUAGE = [{
+  "name": "1",
+  "value": 14
+}, {
+  "name": "2",
+  "value": 18
+}, {
+  "name": "3",
+  "value": 5
+}, {
+  "name": "4",
+  "value": 1
+}];
+
+let LANGUAGE_KNOWN = [{
+  "name": "FRENCH",
+  "value": 8
+}, {
+  "name": "FARSI",
+  "value": 1
+}, {
+  "name": "MANDARIN",
+  "value": 10
+}, {
+  "name": "HINDI",
+  "value": 3
+}, {
+  "name": "PUNJABI",
+  "value": 2
+}, {
+  "name": "TELUGU",
+  "value": 1
+}, {
+  "name": "TAMIL",
+  "value": 1
+}, {
+  "name": "CANTONESE",
+  "value": 2
+}, {
+  "name": "CHINESE",
```

Comment:
i don't know how safe it is to assume but, chinese may be synonymous with mandarin (since canto is explicitly defined?

Reply:
I added this cause people selected CHINESE specifically. I am not sure how else we can represent that information, since I can't assume it's either Mandarin or Cantonese or even both. I am open to suggestions though

Reply:
oo how about "CHINESE (UNSPECIFIED DIALECT)" (or "CHINESE (UNSPECIFIED)"?) or something of the sort? sorry, am being nitpicky about the "chinese is not a single language" angle 😅

Reply:
I like this